tensorboard/plugins/audio/audio_plugin.py

```diff
@@ -57,18 +66,12 @@ def get_plugin_apps(self):
         }
 
     def is_active(self):
-        """The audio plugin is active iff any run has at least one relevant
-        tag."""
-        if not self._multiplexer:
-            return False
-        return bool(
-            self._multiplexer.PluginRunToTagToContent(metadata.PLUGIN_NAME)
-        )
+        return False
```

Comment:
maybe add the ``# `list_plugins` as called by TB core suffices`` comment here as well?

Reply:
Done.